src/activy/connectStrava.py

```
@@ -65,48 +64,58 @@ def connectStravaStage4(device, templates):
     """
     clickNodeByClassInstance(device, "android.widget.Button", 0)
 
     try:
-        tryCheckStage(device, 401, templates)
-        connectStravaStage402(device, templates)
+        stage = tryCheckStage(device, ["4", "401or4o402", "402or4o403"], templates)
+        if stage[0] == "401or4o402":
+            connectStravaStage402(device, templates)
+        elif stage[0] == "402or4o403":
+            connectStravaStage403(device, templates)
     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)